docs/reference/useQuery.md

```diff
@@ -71,20 +71,20 @@ const result = useQuery({
 - `queryKey: unknown[]`
   - **Required**
   - The query key to use for this query.
-  - The query key will be hashed into a stable hash. See [Query Keys](../guides/query-keys) for more information.
+  - The query key will be hashed into a stable hash. See [Query Keys](https://tanstack.com/query/v4/docs/guides/query-keys) for more information.
```

Comment:
I'm not sure what this is all about. These links are working, I just tried them out.

Also, as I said before, we don't want absolute links! That means: no referencing to `https://tanstack.com/query/v4` directly, as that will be very hard to maintain if v5 comes out.

I'm happy that you want to contribute, but please:
- only change links that are not working
- keep them as relative links!
- please only submit one PR that changes all links, not one PR per page - that just unnecessarily spams linboxes.

Thank you